samcli/commands/list/exceptions.py

```diff
@@ -6,7 +6,7 @@
 from samcli.commands.exceptions import UserException
 
 
-class StackOutputsError(UserException):
+class SamListError(UserException):
```

Comment:
Small doc could be useful for this exception. I assume this is a base one that we can use in `sam list` command?

Reply:
Added doc